search-impl/src/main/java/com/example/auction/search/Module.java

```diff
@@ -0,0 +1,30 @@
+package com.example.auction.search;
+
+import com.example.auction.bidding.api.BiddingService;
+import com.example.auction.item.api.ItemService;
+import com.example.auction.search.api.SearchService;
+import com.example.auction.search.impl.BrokerEventConsumer;
+import com.example.auction.search.impl.IndexedStoreImpl;
+import com.example.auction.search.impl.SearchServiceImpl;
+import com.example.elasticsearch.Elasticsearch;
+import com.google.inject.AbstractModule;
+import com.lightbend.lagom.javadsl.server.ServiceGuiceSupport;
+
+/**
+ *
+ */
+public class Module extends AbstractModule implements ServiceGuiceSupport {
+    @Override
+    protected void configure() {
+        bindServices(serviceBinding(SearchService.class, SearchServiceImpl.class));
+
+        bindClient(BiddingService.class);
+        bindClient(ItemService.class);
+        bindClient(Elasticsearch.class);
+
+        bind(IndexedStore.class).to(IndexedStoreImpl.class);
+
+        bind(BrokerEventConsumer.class).asEagerSingleton();
+
```

Comment:
I want to review if there's any difference in running the streams on a Guice `Singleton` vs running them on a Lagom managed `Service`.